auth/resources/authserver_unsafe_attributes.properties

```diff
@@ -0,0 +1,12 @@
+ldapLoginDN=seagate
```

Comment:
add comment here about the default value or see if we can use "" string as empty string

Reply:
Add seagate copyright header

Reply:
use value = dummy